src/index.html

```diff
@@ -35,13 +35,15 @@
   <meta name="msapplication-TileColor" content="#da532c">
   <meta name="msapplication-config" content="assets/favicon/browserconfig.xml">
   <meta name="theme-color" content="#ffffff">
+  <meta property="og:image" content="https://dex.software/assets/images/dex-logo-dark-orange.png"/>
```

Comment:
https://ogp.me/ I read the following:
`og:image:url - Identical to og:image.`
`(...)`
`og:image:alt - A description of what is in the image (not a caption). If the page specifies an og:image it should specify og:image:alt.`

Shouldn't we have the last tag as well?

Reply:
Definitely, I totally looked over this comment!

Updating it rn